subsys/bluetooth/controller/hal/nrf5/ticker.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016-2017 Nordic Semiconductor ASA
+ * Copyright (c) 2016 Vinayak Kariappa Chettimada
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Macro to translate microseconds to tick units.
+ * NOTE: This returns the floor value.
+ */
+#define TICKER_US_TO_TICKS(x) \
```

Comment:
For this and all the macro definitions below: do you think we could align with what we did in hal/nrf5/radio/, i.e. adding a HAL_ prefix that indicates direct HW dependency?

Reply:
Agree.